FirebaseMessaging/Tests/UnitTests/FIRMessagingUtilitiesTest.m

```diff
@@ -69,9 +69,16 @@ - (void)testAppVersionReturnsEmptyStringWhenNotFound {
 
 - (void)testAppIdentifierReturnsExpectedValue {
   NSString *expectedIdentifier = @"com.me.myapp";
+#if TARGET_OS_WATCH
+  expectedIdentifier = @"com.me.myapp.watchkit.watchkitextensions";
+#endif
   [[[_mainBundleMock stub] andReturn:expectedIdentifier] bundleIdentifier];
   NSString *appIdentifier = FIRMessagingAppIdentifier();
+#if TARGET_OS_WATCH
+  XCTAssertEqualObjects(appIdentifier, @"com.me.myapp.watchkit");
```

Comment:
optional: Would it be possible to define a variable for @"com.me.myapp.watchkit" along `expectedIdentifier` to unify the assertion and get rid of the additional `#if TARGET_OS_WATCH`?

Reply:
Done.